src/GitTools.Testing/Fixtures/RemoteRepositoryFixture.cs

```diff
@@ -11,10 +11,7 @@ namespace GitTools.Testing
     public class RemoteRepositoryFixture : RepositoryFixtureBase
     {
         public RemoteRepositoryFixture(Func<string, IRepository> builder)
-            : base(builder)
-        {
-            CreateLocalRepository();
-        }
+            : base(builder) => CreateLocalRepository();
```

Comment:
There is no analyzer to help with this kind of formatting. I will make the changes manually, just let me know your preference between:

1. Always break after `=>`
2. Always break before `=>` (that's the one I usually go for but that's purely personal)
3. Break only on long line.

Reply:
3